subsys/net/lib/aws_iot/src/aws_iot.c

```diff
@@ -1079,6 +1096,16 @@ int aws_iot_init(const struct aws_iot_config *const config,
 		return -ENODATA;
 	}
 
+#if defined(CONFIG_AWS_IOT_BROKER_HOST_NAME_APP)
```

Comment:
If the ifdef guard is removed for `config->host_name` you can format this block the same way its done for `CONFIG_AWS_IOT_CLIENT_ID_APP`.